docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -84,15 +86,18 @@ However, such nodes still have to satisfy the two above requirements.
 
 [[proxy-mode]]
 Proxy mode::
-In proxy mode, a cluster is created using a name and a single proxy address.
-When you register a remote cluster, a configurable number of socket connections
-are opened to the proxy address. The proxy is required to route those
-connections to the remote cluster. Proxy mode does not require remote cluster
-nodes to have accessible publish addresses.
+In proxy mode, a cluster is registered with a name of your choosing and the
+address of a TCP (layer 4) reverse proxy which you must configure to route
```

Comment:
I've read superficially about forward vs reverse proxies, but I don't understand why Elasticsearch would support one and not the other?

Reply:
Oh. Maybe it just clicked. ES sets up a single proxy TCP address. ES just asks that address for everything, the proxy figures out the endpoints. But conceptually a forward proxy wouldn't know about endpoints particularly, it'd be ES specifying different / many endpoints. That's my guess anyway 😬

Reply:
Right yeah I'm not sure "forward TCP proxy" is even meaningful, a forward proxy needs some kind of protocol support so that the client can specify the destination of the proxied connection.